diracx-db/tests/auth/test_authorization_flow.py

```diff
@@ -58,11 +58,6 @@ async def test_insert_id_token(auth_db: AuthDB):
                 uuid, id_token, MAX_VALIDITY
             )
 
-    # We shouldn't be able to retrieve it twice
-    async with auth_db as auth_db:
-        with pytest.raises(AuthorizationError, match="already used"):
-            res = await auth_db.get_authorization_flow(code, MAX_VALIDITY)
-
```

Comment:
Why remove this test?

Reply:
Because:
- I break down `get_authorization_flow` into smaller methods to move the logic from `diracx-db` to `diracx-logic`
- The new `get_authorization_flow` is now very simple and does not require complex testing as it was the case before: complex testing should now be done in `diracx-routers` and it's already done.